pkg/resource/deploy/step.go

```diff
@@ -945,7 +977,17 @@ func (s *ImportStep) Diffs() []resource.PropertyKey                { return s.di
 func (s *ImportStep) DetailedDiff() map[string]plugin.PropertyDiff { return s.detailedDiff }
 
 func (s *ImportStep) Apply(preview bool) (resource.Status, StepCompleteFunc, error) {
-	complete := func() { s.reg.Done(&RegisterResult{State: s.new}) }
+	complete := func() {
+		now := time.Now().UTC()
+		s.new.Updated = &now
```

Comment:
Likewise, an import is not an update. I was assuming we we're treating "updated" as "last time the engine modified the cloud resource" not last time the engine updated the state.

Reply:
The way I originally envisaged this was that the last time we touched the resource or made an API call would be tracked. Updated probably isn't the right semantic for it, neither is "modified" - but the end result really is as a searcher, I really want to know when the last time this particular stack touched or made an API for this particular resource.

Reply:
Maybe, "Last Active" ?

Reply:
@jaxxstorm Can you elaborate what constitutes an API call that merits updating the updated field?

Reply:
Anything in the provider that touches that API, so a created, modified, updated or refreshed (read)

Reply:
@jaxxstorm I'd imagine that this value would be discernible from the manifest timestamp.

On a per resource basis, I think a timestamp tracking the last time that an incongruence between the state and the cloud resource took place would be most useful and in the spirit of the thing you want. In that case, I'd suggest the term `Reconciled`.

Reply:
I suggest starting with only created/modified/updated. If users are surprised that this doesn't also capture refreshes we can consider expanding the field to capture those or track it elsewhere (like "reconciled" as Kyle suggested).